### PR TITLE
[improve] job submit parameter improvement

### PR DESCRIPTION
**Pull request by wolfboys**
[improve] job submit parameter improvement

Currently, when submitting a flink job in streampark, there are three properties, namely `option`, `properties`, and `extraParameter`, which can cause confusion and unclear usage during method invocation, so we have simplified this part by removing the option property and putting all flink The standard configuration of flink is put in `properties`, and the extra parameters are put in `extraParameter`

## What changes were proposed in this pull request

Issue Number: close #xxx 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): no

### CODE REVIEW

**Comment by wolfboys** [comment]
cc @1996fanrui @MonsterChenzhuo PTAL, thanks

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/AppBuildPipeServiceImpl.java [hunk 1]
```diff
@@ -445,16 +445,13 @@ public Map<Long, PipelineStatus> listPipelineStatus(List<Long> appIds) {
             return Maps.newHashMap();
         }
         LambdaQueryWrapper<AppBuildPipeline> queryWrapper = new LambdaQueryWrapper<AppBuildPipeline>()
-            .select(AppBuildPipeline::getAppId, AppBuildPipeline::getPipeStatusCode)
             .in(AppBuildPipeline::getAppId, appIds);
-        List<Map<String, Object>> rMaps = baseMapper.selectMaps(queryWrapper);
+
+        List<AppBuildPipeline> rMaps = baseMapper.selectList(queryWrapper);
```
Comment (muchunjin) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/AppBuildPipeServiceImpl.java]: Should the name of the variable `rMaps` be changed?

**Review by muchunjin — COMMENTED** [review]
@wolfboys Except that the naming of the variable `rMaps` needs to be standardized, I have not found any other problems. I will test it as a whole after the merger

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/AppBuildPipeServiceImpl.java (hunk 1 above)
Comment (wolfboys) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/AppBuildPipeServiceImpl.java]: > @wolfboys Except that the naming of the variable `rMaps` needs to be standardized, I have not found any other problems. I will test it as a whole after the merger

thanks for your review. I will improve what you mentioned

**Comment by wolfboys** [comment]
> @wolfboys Except that the naming of the variable `rMaps` needs to be standardized, I have not found any other problems. I will test it as a whole after the merger

done. thanks for your review

**Comment by MonsterChenzhuo** [comment]
LGTM

**Review by MonsterChenzhuo — APPROVED** [review]

**Pull request merged**